Sources/MongoSwift/WriteConcern.swift

```diff
@@ -131,17 +122,42 @@ public class WriteConcern: Codable {
         }
     }
 
-    /// Initializes a new `WriteConcern` by copying a `mongoc_write_concern_t`.
+    /// Initializes a new `WriteConcern` with the same values as the provided `mongoc_write_concern_t`.
     /// The caller is responsible for freeing the original `mongoc_write_concern_t`.
     internal init(from writeConcern: OpaquePointer?) {
-        self._writeConcern = mongoc_write_concern_copy(writeConcern)
+        if mongoc_write_concern_journal_is_set(writeConcern) {
+            self.journal = mongoc_write_concern_get_journal(writeConcern)
+        } else {
+            self.journal = nil
+        }
+
+       let number = mongoc_write_concern_get_w(writeConcern)
```

Comment:
the alignment looks funky around here, maybe this needs to be one space forward?

Reply:
Updated

Reply:
this still needs to be fixed
```suggestion
        let number = mongoc_write_concern_get_w(writeConcern)
```